File: 프로그래머스/level1/예산.js
function solution(d, budget) {
    var answer = 0;
    d.sort((a,b)=>a-b);
    for(let i=0; i<d.length; i++){
        if(budget>=d[i]){
            answer++;
            budget -= d[i];
        }
        else    break;
    }
    return answer;
}

File: 프로그래머스/level1/예산.cpp
#include <iostream>
#include <stdio.h>
#include <string>
#include <algorithm>
#include <vector>

using namespace std;

int solution(vector<int> d, int budget) {
    int answer = 0;
    vector<int> tmp = d;
    int t = budget;
    sort(tmp.begin(), tmp.end());
    for(int i=0; i<tmp.size(); i++){
        if(t-tmp[i]>=0){
            answer++;
            t -= tmp[i];
        }
    }
    return answer;
}